CHANGES.md

```diff
@@ -1,3 +1,9 @@
+# Version 2.0
+
+* Removed 2 GB stream limit. Extremely large streams can be created by chaining together blocks. Attempts to convert a stream into a contiguous buffer longer than the runtime's limits will result in an exception.
+* Removed build targets for net40, net462, and netstandard1.4.
```

Comment:
1. Technically, you removed `net46` instead of `net462`. I'm not that familiar with how much the build system cares about that third part of the .NET Framework version - is this an important distinction to make?
1. Maybe mention that we still build for net45 - this sentence alone would kind of make me suspect that there's no longer support for applications on .NET 4.6.2.
    1. (though even if we didn't have a net45 build target, applications using .NET Framework 4.6.2 would still be able to depend on the netstandard2.0 build. But that's probably not worth mentioning in the change log, people can figure that out themselves).